src/components/Comments/ReplyCommentItem.js

```diff
@@ -38,6 +47,7 @@ const ReplyCommentItem = ({
   updateVoteParent,
   onLongPress
 }) => {
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refSheet` is introduced here but it's not clear what it's used for. It would be beneficial to add a comment explaining its purpose.
```diff
+  // refSheet is used to control the BottomSheetMenu component
   const refSheet = React.useRef();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->